packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -110,11 +117,15 @@ async def subscribe(
             # consumer/publisher must set the same configuration for same queue
             # exclusive means that the queue is only available for THIS very client
             # and will be deleted when the client disconnects
-            queue = await channel.declare_queue(
+            queue_parameters = dict(
                 durable=True,
-                exclusive=True,
+                exclusive=exclusive_queue,
                 arguments={"x-message-ttl": _RABBIT_QUEUE_MESSAGE_DEFAULT_TTL_S},
             )
+            if not exclusive_queue:
+                # NOTE: setting a name will ensure multiple instance will take their data here
+                queue_parameters |= {"name": exchange_name}
```

Comment:
I think above is way too fancy for one single element.
```suggestion
                queue_parameters["name"] = exchange_name
```

Reply:
I like fancy. and it's only the beginning...